README.md

```diff
@@ -75,6 +75,252 @@ style('btn')->apply('p-4 bg-blue text-white');
+    </p>
+HTML);
+```
+
+### `<b>` and `<strong>`
+
+The `<b>`and `<strong>` elements can be used to mark the text as **bold**.
+
+**Default Styles**: `font-bold`
+
+```php
+render(<<<'HTML'
+    <p>
+        This is a CLI app built with <b>Termwind</b>.
+    </p>
+HTML);
+```
+
+### `<i>` and `<em>`
+
+The `<i>`and `<em>` elements can be used to mark the text as *italic*.
+
+**Default Styles**: `italic`
+
+```php
+render(<<<'HTML'
+    <p>
+        This is a CLI app built with <i>Termwind</i>.
+    </p>
+HTML);
+```
+
+### `<br>`
+
+The `<br>` element can be used to do a line break.
+
+```php
+render(<<<'HTML'
+    <p>
+        This is a CLI <br>
+        app built with Termwind.
+    </p>
+HTML);
+```
+
+### `<ul>`
+
+The `<ul>` element can be used for an unordered list. It can only accept `<li>` elements as childs, if there is another element provided it will throw an `InvalidChild` exception. 
+
+**Default Styles**: `block`, `list-disc`
+
+```php
+render(<<<'HTML'
+    <ul>
+        <li>Item 1</li>
+        <li>Item 2</li>
+    </ul>
+HTML);
+```
+
+### `<ol>`
+
+The `<ol>` element can be used for an ordered list. It can only accept `<li>` elements as childs, if there is another element provided it will throw an `InvalidChild` exception. 
+
+**Default Styles**: `block`, `list-decimal`
+
+```php
+render(<<<'HTML'
+    <ol>
+        <li>Item 1</li>
+        <li>Item 2</li>
+    </ol>
+HTML);
+```
+
+### `<li>`
+
+The `<li>` element can be used as a list item. It should only be used as a child of `<ul>` and `<ol>` elements.
+
+**Default Styles**: `block`, `list-decimal`
+
+```php
+render(<<<'HTML'
+    <ul>
+        <li>Item 1</li>
+    </ul>
+HTML);
+```
+
+### `<dl>`
+
+The `<dl>` element can be used for an description list. It can only accept `<dt>` or `<dd>` elements as childs, if there is another element provided it will throw an `InvalidChild` exception. 
+
+**Default Styles**: `block`
+
+```php
+render(<<<'HTML'
+    <dl>
+        <dt>🍃 Termwind</dt>
+        <dd>Give your CLI apps a unique look</dd>
+    </dl>
+HTML);
+```
+
+### `<dt>`
+
+The `<dt>` element can be used as a description title. It should only be used as a child of `<dl>` elements.
+
+**Default Styles**: `block`, `font-bold`
+
+```php
+render(<<<'HTML'
+    <dl>
+        <dt>🍃 Termwind</dt>
+    </dl>
+HTML);
+```
+
+### `<dd>`
+
+The `<dd>` element can be used as a description title. It should only be used as a child of `<dl>` elements.
+
+**Default Styles**: `block`, `ml-4`
+
+```php
+render(<<<'HTML'
+    <dl>
+        <dd>Give your CLI apps a unique look</dd>
+    </dl>
+HTML);
+```
+
+### `<hr>`
+
+The `<hr>` element can be used as an horizontal line.
+
+```php
+render(<<<'HTML'
+    <div>
+        <div>🍃 Termwind</div>
+        <hr>
+        <p>Give your CLI apps a unique look</p>
+    </div>
+HTML);
+```
+
+### `<table>`
+
+The `<table>` element can have columns and rows.
+
+```php
+render(<<<'HTML'
```

Comment:
Look at full example for table element. I think it's important to show all features

```html
<table style="box-double">
    <thead title="Books">
        <tr>
            <th align="right">ISBN</th>
            <th>Title</th>
            <th>Author</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <th align="right">99921-58-10-7</th>
            <td>Divine Comedy</td>
            <td align="right">Dante Alighieri</td>
        </tr>
        <tr border="1">
            <th class="bg-blue text-red" align="right">9971-5-0210-0</th>
            <td>A Tale of Two Cities</td>
            <td align="right">Charles Dickens</td>
        </tr>
        <tr>
            <th align="right">960-425-059-0</th>
            <td>The Lord of the Rings</td>
            <td align="right">J. R. R. Tolkien</td>
        </tr>
        <tr>
            <th align="right">80-902734-1-6</th>
            <td>And Then There Were None</td>
            <td rowspan="2" align="right">Dante Alighieri<br>spans multiple rows</td>
        </tr>
        <tr>
            <th align="right">978-052156781</th>
            <td>De Monarchia</td>
        </tr>
    </tbody>
    <tfoot title="Page 1/2">
        <tr>
            <th colspan="3">This value spans 3 columns.</th>
        </tr>
    </tfoot>
</table>
```

And also tell about table styles
 - default
 - borderless
 - compact
 - box
 - box-double
 - symfony-style-guide

Reply:
And mention table cells can't have inner elements, only text

Reply:
Yes the table styles are unique.